### PR TITLE
Cwl browse

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**
This is first week progress

### CODE REVIEW

**Comment by stain** [comment]
Looks good! You are adding cwl-logo-header.png, which has an [incompatible license](https://github.com/common-workflow-language/logo/blob/master/LICENSE.md).

Could you remove that, and I'll raise with CWL to change the license.

**Comment by stain** [comment]
Ah now, the CC SA for unmodifiable binaries, so OK as long as we don't modify the CWL logo. 
http://www.apache.org/legal/resolved.html#cc-sa

We just need to add that copyright statement to our NOTICE.

File: taverna-cwl-activity-ui/src/main/java/org/apache/taverna/cwl/ui/serviceprovider/CwlServiceProviderConfig.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.apache.taverna.cwl.ui.serviceprovider;
```
Comment (stain) [inline taverna-cwl-activity-ui/src/main/java/org/apache/taverna/cwl/ui/serviceprovider/CwlServiceProviderConfig.java:17]: Could you add a license header also here? :)

Tip: run

```
mvn apache-rat:check
```

on your module(s) to see if you forgot a file.

File: taverna-cwl-activity-ui/src/test/java/org/apache/taverna/cwl/ui/TestWorkBench.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.apache.taverna.cwl.ui;
```
Comment (stain) [inline taverna-cwl-activity-ui/src/test/java/org/apache/taverna/cwl/ui/TestWorkBench.java:17]: and here..

Comment (ThilinaManamgoda) [inline taverna-cwl-activity-ui/src/test/java/org/apache/taverna/cwl/ui/TestWorkBench.java:17]: done

**Comment by stain** [comment]
Thank you! Merged into `cwl-browse` branch.

**Pull request merged**